frontend_tests/node_tests/widgetize.js

```diff
@@ -0,0 +1,178 @@
+set_global('$', global.make_zjquery());
+set_global('voting_widget', {});
+set_global('tictactoe_widget', {});
+set_global('zform', {});
+set_global('document', 'document-stub');
+
+let narrow_deactivated_callback;
+$(document).on = (selector, func) => {
```

Comment:
Not sure about this approach of capturing callback, but I had to do this since `$(document).on` is global in `widetize.js`